lib/web_ui/lib/src/engine/canvaskit/image_web_codecs.dart

```diff
@@ -122,6 +127,34 @@ void _bgrToRgba(ByteBuffer pixels) {
     // codecs that do something different.
     pixelBytes[i] = r;
     pixelBytes[i + 2] = b;
+    if (isBgrx) {
+      pixelBytes[i + 3] = 255;
+    }
+  }
+}
+
+/// Based on Chromium's SetRGBAPremultiply.
+int _premultiply(int value, int alpha) {
```

Comment:
It might make sense to inline this function using `@pragma('dart2js:tryInline')` as images can be pretty big and could take advantage of faster pixel processing logic.

Reply:
Added the inlining annotation